libs/blocks/modal/modal.css

```diff
@@ -21,6 +21,10 @@
   z-index: 102;
 }
 
+.dialog-modal.commerce-frame {
+  z-index: 110;
```

Comment:
I think we can safely use `103` in this case.

Reply:
done.